bench/simple_consumer_bench.py

```diff
@@ -0,0 +1,111 @@
+import json
```

Comment:
Having some standard benchmarking code around is helpful. I say clean up and keep.